tests/testthat/testdata/create_double_data.R

```diff
@@ -0,0 +1,29 @@
+# yaml workflow
+test_wf <- MakeWorkflowList(strPath = test_path("testdata"), strNames = "test_workflow")
+test_mapping <- MakeWorkflowList(strPath = test_path("testdata"), strNames = "mapping")
+lRaw <- UseClindata(
```

Comment:
This (and the dfInput below) somewhat blindly uses clindata, because that's what it was when I got this PR. Should this be using some sampleData, or do we want to continue from raw? @lauramaxwell @jwildfire

Reply:
let's continue from Raw, so we can test the full yaml workflow (including `Input_Rate()`)